src/index.ts

```diff
@@ -29,6 +29,62 @@ const resolveHeaders = (headers: Dom.RequestInit['headers']): Record<string, str
   return oHeaders
 }
 
+/**
+ * Fetch data using POST method
+ */
+const post = async <V = Variables>(
+  url: string,
```

Comment:
I followed the way you declare functions to keep consistency but I'm not a huge fan of using too much args here and I spend much time to think if I should use a typed object here. Tell me if it is okay for you or if you want me to change this